### DISCUSSION TITLE
Need a programme that makes the turtle follow the player.

### DISCUSSION THREAD

**Discussion by HenryDu8133** [cc-tweaked/CC-Tweaked#1938] [Q&A]
Hi there! Can someone help me write a program to make the turtle follow the player? Thank you!

**Comment by tonitch** [comment]
Help me write as you already have a base that you can show us or as in doing in your stead ?

**Comment by piprett** [comment]
Instead of asking us to spoon feed you, can you specify what you need help with? Where are you stuck?
